### DISCUSSION TITLE
0.15 Beta 1 - Add detection percent to explore or bring frigate+ back

### DISCUSSION THREAD

**Discussion by tazzytazzy** [blakeblackshear/frigate#14987] [Ideas]
I'm  using 0.15 beta 1.

I used to use the frigate+ section to quickly review the detection percent. If it was lower than 80%, I would be able to quickly upload that image to frigate+ to better train the model.

Now, there is no fast and easy way to quickly view the detection percent for images.

Could it be possible to overlay the the % number on the explore page just like it displays when exploring a specific label?

I've been a frigate+ supporter since it first became available as I want to support this amazing project. Thanks for you all your hardwork.

**Comment by hawkeye217** [comment]
The detection percentage is still there in the grid view of Explore, are you just looking at the summary view?

**Comment by hawkeye217** [comment]
It's also worth mentioning that the next beta will add the ability to filter snapshots based on their "submitted to Frigate+" status. That, combined with the score filter and arrow key navigation, should make submitting to Frigate+ much faster than the previous version.

**Reply by mimainka** [reply]
The filtering option regarding Frigate+ is so good news :)
